### PR TITLE
Use the new "type" labels after GH migration.

### PR DESCRIPTION
**Pull request by Mariatta**
Closes https://github.com/python/core-workflow/issues/432

### CODE REVIEW

File: bedevere/prtype.py [hunk 1]
```diff
@@ -4,18 +4,19 @@
...
-    performance = f"{LABEL_PREFIX}-performance"
-    security = f"{LABEL_PREFIX}-security"
-    tests = f"{LABEL_PREFIX}-tests"
+
+    type_bug = f"{TYPE_LABEL_PREFIX}-bug"
+    documentation = "docs"
+    type_feature = f"{TYPE_LABEL_PREFIX}-feature"
+    performance = "performance"
+    type_security = f"{TYPE_LABEL_PREFIX}-security"
+    tests = "tests"
```
Comment (ezio-melotti) [inline bedevere/prtype.py:19]: `type-crash` is missing from the list, but it probably doesn't matter.

**Review by ezio-melotti — COMMENTED** [review]
LGTM.

Do you think there's any downside applying it now or is it better/safer to wait?  I can still update the labels on my end so that they match the new labels during the migration.

On a more general note, many of the new labels are related to specific dirs/files in the repo, so bevedere will probably need some refactoring so that it can add more labels (also because `docs`, `performance`, `tests` are no longer types).

File: bedevere/prtype.py (hunk 1 above)
Comment (Mariatta) [reply bedevere/prtype.py:19]: Yeah I think bedevere really only cares about the `docs` and `tests` types at this moment. It doesn't do anything for the other type- labels.

**Comment by Mariatta** [comment]
I would suggest changing the labels first on the CPython repo, and then deploy bedevere afterwards (by merging this PR).

**Comment by ezio-melotti** [comment]
That makes sense, should we do it now?

**Comment by Mariatta** [comment]
Yup let's do it!

**Comment by ezio-melotti** [comment]
Thanks for the PR!
I renamed the labels on `python/cpython` and merged it.

**Pull request merged**